### PR TITLE
Consortia reporting

### LINKED ISSUE #153: Consortia reporting
Clarify section 10.3, COUNTER reporting for consortia, using rules from https://www.projectcounter.org/about/reporting-counter-usage-for-consortia/

### PR DESCRIPTION
**Pull request by tashalouizamc**
Updated section 10.3 to import details from counter website  https://www.projectcounter.org/about/reporting-counter-usage-for-consortia/

### CODE REVIEW

File: source/10-compliance/03-counter-reporting-for-consortia.rst [hunk 1]
```diff
@@ -7,38 +7,52 @@
...
+"""""""""""""""""""""""""""""""
 
-Content providers MUST support the /members COUNTER_SUSHI API path to provide the consortium with the list of their members on the platform and the SUSHI credentials for each member. This will enable tools to be created to efficiently retrieve member usage and create separate or consolidated reporting.
+There are three types of consortia-level reporting:
 
+* Report providers MUST offer summary reports that show total usage for all members of the consortium rolled up to the consortia level, and which cannot be broken down by institution. The totals on the summary report may differ from the sum of the totals on individual member reports for the same items if an authentication method used identifies to multiple member sites and usage it attributed to each such site (e.g. overlapping IP ranges).
+* Report providers may elect to offer detailed reports that show total usage for all members of the consortium rolled up to the consortia level, and which can be broken down by institution. As with the summary report, the totals on the detailed report may differ from the sum of the totals on individual member reports for the same items.
```
Comment (berndoberknapp) [inline source/10-compliance/03-counter-reporting-for-consortia.rst]: I don't understand this option - how can a report both be rolled up at the consortia (or consortium?) level and broken down by institution?

File: source/10-compliance/03-counter-reporting-for-consortia.rst [hunk 2]
```diff
@@ -7,38 +7,52 @@
...
+* Institutional reports are the individual institutional-level reports. Consortia managers and administrators MUST be able to retrieve institutional reports.
 
-Privacy and Confidentiality
-"""""""""""""""""""""""""""
+Note that consortia-level reports are restricted to COUNTER Reports. The Standard Views of the COUNTER Reports are not available as consortia-level reports.
```
Comment (berndoberknapp) [inline source/10-compliance/03-counter-reporting-for-consortia.rst]: The Standard Views are required in 5.0 for the summary reports. Why would we restrict this to the COUNTER Reports in 5.1?

File: source/10-compliance/03-counter-reporting-for-consortia.rst [hunk 3]
```diff
@@ -7,38 +7,52 @@
...
 
-When a COUNTER report is harvested by a consortium administrator, a content provider may choose to limit member usage to include only content acquired through the consortium. Note that when such a limitation is in place the resulting report may differ from the member-sites own version of the report. Since not all content providers can provide such limits, the consortium will be responsible for ensuring usage is filtered to the content they license for members.
+* The Customer_ID used in a detailed report MUST match that used in a report to the individual institution, and the ID for the institution returned by the /members COUNTER_SUSHI API path.
+* If provided, the Institution_Name used in a detailed report should match that used in a report to the individual institution, and the name for the institution returned by the /members COUNTER_SUSHI API path.
+* If provided, Institution_Name should be the first column in the tabular report, with Customer_ID the second column. Otherwise Customer_ID should be the first column.
```
Comment (berndoberknapp) [inline source/10-compliance/03-counter-reporting-for-consortia.rst]: These are MUST rules according to Section 11.5.

File: source/10-compliance/03-counter-reporting-for-consortia.rst [hunk 4]
```diff
@@ -7,38 +7,52 @@
...
+* The Customer_ID used in a detailed report MUST match that used in a report to the individual institution, and the ID for the institution returned by the /members COUNTER_SUSHI API path.
+* If provided, the Institution_Name used in a detailed report should match that used in a report to the individual institution, and the name for the institution returned by the /members COUNTER_SUSHI API path.
+* If provided, Institution_Name should be the first column in the tabular report, with Customer_ID the second column. Otherwise Customer_ID should be the first column. 
 
-When the content provider chooses to limit member usage to only content acquired through the consortium, they MUST include a message to this effect in the Notes element in their implementation of the /members path in the COUNTER_SUSHI API (see :numref:`sushi` above).
```
Comment (berndoberknapp) [inline source/10-compliance/03-counter-reporting-for-consortia.rst:30]: In my opinion this is an important rule that we should keep. We could either keep it in this section or move it to the /members description in Section 8.1.

**Review by berndoberknapp — CHANGES_REQUESTED** [review]
Please see the inline comments. Note that I've pushed some minor fixes, so you have to update your clone before making further changes.

File: source/10-compliance/03-counter-reporting-for-consortia.rst (hunk 3 above)
Comment (tashalouizamc) [reply source/10-compliance/03-counter-reporting-for-consortia.rst]: I thought MUST wasn't being applied to if/or situations?

File: source/10-compliance/03-counter-reporting-for-consortia.rst (hunk 4 above)
Comment (tashalouizamc) [reply source/10-compliance/03-counter-reporting-for-consortia.rst:30]: The line wasn't deleted, just moved to line 56 under the heading content to report usage on

File: source/10-compliance/03-counter-reporting-for-consortia.rst (hunk 3 above)
Comment (berndoberknapp) [reply source/10-compliance/03-counter-reporting-for-consortia.rst]: Providing the reports is optional, but we expect report providers to follow the rules for reserved extensions if they use them (and the general rules for all COUNTER reports), same as we expect a report provider that optionally delivers IR to follow the rules for IR. If they don't want to follow the rules for the reserved extensions, they could create their own proprietary extensions, but that would make things more difficult for consortia that want to use the reports.

**Review by berndoberknapp — APPROVED** [review]

**Pull request merged**